E-commerce/products.html

```diff
@@ -11,132 +11,158 @@
 </head>
 
 <body>
-  <!-- nav bar -->
-  <div class="topnav">
-    <a class="active" href="index.html">Home</a>
-    <a href="products.html">Products</a>
-    <a href="contact.html">Contact</a>
-  </div>
-  <!-- products -->
-  <main>
-    <section class="best-selling">
-      <h1>Laser Engraved Items</h1>
-      <div class="grid-container">
-        <div class="flex"><img
-            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTEnzhBTtcJOAlFHY2WtkpBf7oIdM9MLqaSDg&usqp=CAU"
-            alt="laser engraved box">
```

Comment:
This is where the page was breaking. This image didn't have the container around it.